src/site/content/en/progressive-web-apps/manifest-updates/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: How Chrome handles updates to the web app manifest
+subhead: What it takes to change icons, shortcuts, colors, and other metadata for your PWA
+authors:
+  - petelepage
+date: 2020-10-12
+updated: 2020-10-12
+description: What it takes to change icons, shortcuts, colors, and other metadata in your web app manifest for your PWA.
+tags:
+  - progressive-web-apps
+---
+
+{% Aside %}
+We are currently gathering data on browsers other than Chrome. If you would
+like to help us gather this data or add content to this page, please leave a
+comment in [issue #4038](https://github.com/GoogleChrome/web.dev/issues/4038).
+{% endAside %}
+
+When a PWA is installed, the browser uses information from the web app
+manifest for the app name, the icons the app should use, and the URL that
+should be opened when the app is launched. But what if you need to update
+your app name, or provide new icons? When and how are those changes reflected
+in the browser?
+
+{% Aside 'caution' %}
+Do not change the name or location of your web app manifest file, doing so
+may prevent the browser from updating your PWA.
+{% endAside %}
+
+In most cases, changes should be reflected within a day or two of the
+manifest being updated, and the PWA subsequently launched.
```

Comment:
You mean the first launch within a day or two of the manifest being updated, right? I still think this could be clearer.

Reply:
How about: "In most cases, changes should be reflected within a day or two of the
PWA being launched after the manifest has been updated."

Reply:
Yes!

Reply:
Great - will update momentarily.